extensions/PhabBugz/lib/WebService.pm

```diff
@@ -89,19 +90,13 @@ sub revision {
     }
     # else bug is private
     else {
-        my $phab_sync_groups = Bugzilla->params->{phabricator_sync_groups}
-            || ThrowUserError('invalid_phabricator_sync_groups');
-        my $sync_group_names = [ split('[,\s]+', $phab_sync_groups) ];
-
-        my $bug_groups = $bug->groups_in;
-        my $bug_group_names = [ map { $_->name } @$bug_groups ];
-
-        my @set_groups = intersect($bug_group_names, $sync_group_names);
+        my @set_groups = get_security_sync_groups($bug);
 
         # If bug privacy groups do not have any matching synchronized groups,
         # then leave revision private and it will have be dealt with manually.
         if (!@set_groups) {
-            ThrowUserError('invalid_phabricator_sync_groups');
+            #ThrowUserError('invalid_phabricator_sync_groups');
+            add_security_sync_comments(@revisions, $bug);
         }
```

Comment:
We can just let this fall through, right?  An admin policy doesn't hurt anything, right?  Or do we need to do something else?